test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -91,13 +94,38 @@
     /**
      * Convert the entity from a {@link Response} into a map of maps.
      */
-    public Map<String, Object> entityAsMap(Response response) throws IOException {
+    public static Map<String, Object> entityAsMap(Response response) throws IOException {
         XContentType xContentType = XContentType.fromMediaTypeOrFormat(response.getEntity().getContentType().getValue());
-        try (XContentParser parser = createParser(xContentType.xContent(), response.getEntity().getContent())) {
+        // EMPTY and THROW are fine here because `.map` doesn't use named x content or deprecation
+        try (XContentParser parser = xContentType.xContent().createParser(
+                NamedXContentRegistry.EMPTY, DeprecationHandler.THROW_UNSUPPORTED_OPERATION,
+                response.getEntity().getContent())) {
             return parser.map();
         }
     }
 
+    /**
+     * Does the cluster being tested have xpack installed?
+     */
+    public static boolean hasXPack() throws IOException {
```

Comment:
I made this `static` so that we could use it easily from `feature`. I feel like if `client()` is static then this can be.